ups/eupspkg.cfg.sh

```diff
@@ -0,0 +1,6 @@
+build(){
+    scons opt=3 prefix=$PREFIX version=$VERSION
+}
+install(){
+    scons opt=3 install prefix=$PREFIX version=$VERSION
+}
```

Comment:
These shouldn't be necessary -- you should be able to completely remove the `eupspkg.cfg.sh` file; eupspkg already knows how to build LSST packages that use `scons`.